src/routes/category/[category]/+page.svelte

```
@@ -0,0 +1,89 @@
+    );
+
+    // Filter articles to only include those that match the formattedCategory
+    const filteredArticles = $derived(
+        articles.filter(article =>
+            article.categories.some(cat =>
+                cat.name.toLowerCase() === formattedCategory.toLowerCase()
+            )
+        )
+    );
+
+    let currentPage = 1;
```

ESLint

[error] `currentPage` is updated, but is not declared with `$state(...)`. Changing its value will not correctly trigger updates(non_reactive_update)

(svelte/valid-compile)